sources/packages/web/src/components/institutions/locations/coe/COEApprovalDisabledBanner.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <banner
+    v-if="
+      coeApprovalPeriodStatus !== COEApprovalPeriodStatus.WithinApprovalPeriod
+    "
+    class="mb-2"
+    :type="BannerTypes.Warning"
+    :header="header"
+    :summary="summary"
+  />
+</template>
+
+<script lang="ts">
+import { PropType, computed, defineComponent } from "vue";
+import { COEApprovalPeriodStatus, BannerTypes } from "@/types";
+
+export default defineComponent({
+  props: {
+    coeApprovalPeriodStatus: {
+      type: Object as PropType<COEApprovalPeriodStatus>,
+      required: true,
+    },
+  },
+  setup(props) {
+    const header = computed<string>(() => {
+      if (
+        props.coeApprovalPeriodStatus ===
+        COEApprovalPeriodStatus.BeforeApprovalPeriod
+      ) {
+        return "Enrolment is not available to confirm yet";
+      }
+      if (
+        props.coeApprovalPeriodStatus ===
+        COEApprovalPeriodStatus.AfterApprovalPeriod
+      ) {
+        return "Enrolment cannot be confirmed ";
+      }
+      return "";
+    });
+
+    const summary = computed<string>(() => {
+      if (
+        props.coeApprovalPeriodStatus ===
+        COEApprovalPeriodStatus.BeforeApprovalPeriod
+      ) {
+        return (
+          "This application will be required to confirm enrolment," +
+          " however it is still outside of the 21 day window before" +
```

Comment:
Does this 21 day window can change in the future? Should it be configured in the application?

Reply:
This value if needs to come from variable, we need to add that to API and use here. This effort may not be required now and can be done in future if required.

Reply:
Are you going to create a ticket for that?